package/addons-update.sh

```diff
@@ -51,13 +51,13 @@ metadata:
     addonmanager.kubernetes.io/mode: EnsureExists
 EOF
 
-GCR_IO_REGISTRY=${REGISTRY:-gcr.io}
+GCR_IO_REGISTRY=${REGISTRY:-docker.io}
```

Comment:
@hongxiaolu lets rename the parameter to align with the default registry name. Renaming would require changing the references in addon manifests.

Reply:
@hongxiaolu it looks like the docker.io param already exists https://github.com/rancher/kubernetes-package/blob/k8s-v1.9/package/addons-update.sh#L55

so it just needs to be replaced in the add-ons yamls as well as in the catalog PR.

Reply:
we should also remove any reference to the GCR registry to ensure we didn't miss anything.